can/tests/test_can_parser.py

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env python3
+import glob
+import os
+import time
+import unittest
+
+from opendbc import DBC_PATH
+from opendbc.can.parser import CANParser
+
+
+class TestCANParser(unittest.TestCase):
+  @classmethod
+  def setUpClass(cls):
+    cls.dbcs = {}
+    for dbc in glob.glob(f"{DBC_PATH}/*.dbc"):
+      dbc_name = os.path.basename(dbc).split('.')[0]
+      with open(dbc, 'r') as f:
+        cls.dbcs[dbc_name] = len(f.readlines())
+
+  def test_dbc_parsing_speed(self):
```

Comment:
let's also add a check for the overall time that's slightly more than the current max

Reply:
Sometimes on my PC the ford DBC can take 80 ms, but usually it's 30 ms